pkg/controller/direct/monitoring/monitoringdashboard_controller.go

```diff
@@ -90,12 +90,12 @@ func (m *dashboardModel) AdapterForObject(ctx context.Context, kube client.Reade
 	if err != nil {
 		return nil, err
 	}
-	projectID := projectRef.ProjectID
-	if projectID == "" {
+	if projectRef == nil || projectRef.ProjectID == "" {
```

Comment:
I think we will never hit `projectRef == nil`(?) [code](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/pkg/controller/direct/references/projectref.go#L35) , or maybe this infers you want to change the `ResolveProject`? Not a blocker, just curious about this change.

Reply:
True.  I'm not sure why I did it.  Reverting.